include/cantera/equil/MultiPhase.h

```diff
@@ -115,12 +115,16 @@ class MultiPhase
     }
 
     //! Check that the specified element index is in range.
-    //! Throws an exception if m is greater than nElements()-1
-    void checkElementIndex(size_t m) const;
+    /*!
+     * @since After %Cantera 3.2, returns verified element index.
+     * @exception Throws an IndexError if m is greater than nElements()-1
```

Comment:
```suggestion
     * @since Starting in %Cantera 3.2, returns the input element index, if valid.
     * @exception Throws an IndexError if m is greater than nElements()-1
```